test/e2e/gardener/shoot/gardenerupgrade/create_hibernate_upgrade_wakeup_delete.go

```diff
@@ -0,0 +1,47 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
General remark / discussion: We could start to remove the year from the copyright header from now on.

```suggestion
// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
```